default.nix

```diff
@@ -5,6 +5,14 @@ python3.pkgs.buildPythonApplication rec {
   name = "nixpkgs-review";
   src = ./.;
   buildInputs = [ makeWrapper ];
+
+  propagatedBuildInputs = [
+    python3.pkgs.humanize
```

Comment:
Eventually I want get rid of humanize/backport again, but I can also do this later on master.
Right now we are unsetting PYTHONPATH to avoid injecting dependencies into nix-shell.

Reply:
Do you prefer if I vendor the `naturaldelta` function from `humanize` into this PR, or we could also do something based on wrappers so that we get `python3.pkgs.humanize` on sys.path even though external settings of PYTHONPATH are ignored?

Reply:
Vendoring the code is probably less hacky. Maybe we can also rewrite the function if it is less complex.